helm_express/helm-express/templates/deployments/redis.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  labels:
+    app: redis
+  name: redis
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: redis
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: redis
+    spec:
+      containers:
+        - image: {{ .Values.redis.image }}
+          name: redis
+          ports:
+            - containerPort: {{ .Values.redis.containerPort }}
```

Comment:
About deployment: I'm using Helm chart (a k8s package manager) to deploy.
It's pretty much the same as k8s. Furthermore, it does string interpolation for k8s template.